setup.py

```diff
@@ -1,4 +1,27 @@
-from setuptools import setup, find_packages
+# -*- coding: utf-8 -*-
+# Copyright IRT Antoine de Saint Exupéry et Université Paul Sabatier Toulouse III - All
```

Comment:
As said in the MIT licence, I believe that All rights reserved explicitly means that you cannot use it in the context of the MIT licence

Reply:
"When you do some work, the default copyright position is all rights reserved. Nobody can do anything with your work without your express permission, or a license to do so.

The MIT license is your license to other people, to make it clear what they can do. However, any rights not explicitly granted by the license are still reserved by you.

Since the default position is all rights reserved, it's not strictly necessary to add that to a copyright notice - but it does make it clearer, and can help avoid scenarios where someone uses your work illegally.

By having the all rights reserved notice, and then a license, you're affirming it's your work and your rights, but you allow people to do these specific things with it."

Reply:
Ok seems more clear to me :)